src/components/atoms/HomeArrow.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+	<div v-on:click="$emit('traverse', arrow.direction)" class="home__arrow-outline">
```

Comment:
why emit ccustum event from button when you can put onclick event directly on button component?

Reply:
Because logic is written in the organism, so I emit custom event to organism and then run my method there